src/bijectors/simplex.jl

```diff
@@ -237,16 +222,10 @@ function simplex_link_jacobian(
                 ((one(T) + ϵ) - sum_tmp)^2
         end
     end
-    @inbounds sum_tmp += x[K - 1]
-    @inbounds if !proj
-        @simd for i in 1:K
-            dydxt[i, K] = -1
-        end
-    end
-    return UpperTriangular(dydxt)'
+    return dydxt'
```

Comment:
The adjoint operation seems a bit annoying but I guess the algorithm should be updated in separate PRs if desired.

Reply:
Likewise, agreed, but this should be fixed separately.